### PR TITLE
Move ConstantFloat class factory to separate file

### LINKED ISSUE #29: Remove `cgp_constant_float` factory
The factory seems pretty pointless (https://github.com/jakobj/python-gp/blob/master/gp/cgp_node.py#L236) and should be removed: it takes only three lines to define your own constant float node (and avoiding pickling issues):
```python
class CustomCGPConstantFloat(CGPConstantFloat):
    def __init__(self, idx, inputs):
      super().__init__(idx, inputs)
      self._output = val
```
isn't that short enough? @mschmidt87

### PR DESCRIPTION
**Pull request by jakobj**
closes #29

### CODE REVIEW

File: test/test_cgp_node.py [hunk 1]
```diff
@@ -71,12 +71,10 @@ def test_mul():
...
 
 
-def test_const_float():
+def test_constant_float():
     params = {"n_inputs": 2, "n_outputs": 1, "n_columns": 1, "n_rows": 1, "levels_back": 1}
 
-    val = 1.678
-
-    primitives = [gp.custom_cgp_constant_float(val)]
+    primitives = [gp.CGPConstantFloat]
```
Comment (mschmidt87) [inline test/test_cgp_node.py]: Minor: Add a small comment explaining that this constant node has value 1.

File: test/test_cgp_node.py [hunk 2]
```diff
@@ -88,19 +86,7 @@ def test_const_float():
     genome.dna = [-1, None, -1, None, 0, None, -2, 2]
     graph = gp.CGPGraph(genome)
 
-    x = [None, None]
+    x = [5.0, 1.5]
```
Comment (mschmidt87) [inline test/test_cgp_node.py]: Why not keep using `None` here?

Comment (jakobj) [reply test/test_cgp_node.py]: ok

**Comment by jakobj** [comment]
github <-> travis comm == bork, merging

**Pull request merged**